Jenkinsfile

```
@@ -11,7 +11,7 @@ pipeline {
   parameters {
     string(name: 'EXTERNALS', defaultValue: 'devAdePT/latest', description: 'LCG software stack in CVMFS')
     choice(name: 'MODEL', choices: ['experimental', 'nightly', 'continuous'], description: 'CDash model')
-    choice(name: 'COMPILER', choices: ['gcc8', 'gcc10', 'clang10', 'native'])
+    choice(name: 'COMPILER', choices: ['gcc8', 'gcc10', 'gcc11', 'clang10', 'native'])
```

```suggestion
    choice(name: 'COMPILER', choices: ['gcc11', 'gcc8', 'gcc10', 'clang10', 'native'])
```

The first value is the default